requirements.txt

```diff
@@ -13,6 +13,7 @@ inflect==5.3.0
 ipython>=7.31.1
 librosa==0.8.1
 matplotlib==3.5.1
+nltk==3.7

```

Comment:
This import is being used in testing to determine the similarity between two sentences.